src/telemetry/telemetry.h

```diff
@@ -21,17 +21,18 @@
 #define TELEMETRY_HOST "telemetry.timescale.com"
 #define TELEMETRY_PATH "/v1/metrics"
 
+#define	NUM_VERSION_CHARS_TO_PRINT	6
```

Comment:
Considering this isn't enough to print versions we've used (`1.0.0-rc3` is 9 chars) I think we should use something longer here; `128` seem completely reasonable to me.